#goal definition
float32 height
---
#result definition
float32 heightDifference
---
#feedback
float32 heightCurrent
